cmd/fleet/server_integration_test.go

```diff
@@ -268,9 +266,10 @@ func TestServerInstrumentation(t *testing.T) {
 	cli := cleanhttp.DefaultClient()
 	callCheckinFunc := func() {
 		for {
-			agentId := "1e4954ce-af37-4731-9f4a-407b08e69e42"
-			cli.Post(srv.buildUrl(agentId, "checkin"), "application/json", bytes.NewBuffer([]byte("{}")))
+			agentID := "1e4954ce-af37-4731-9f4a-407b08e69e42"
+			res, err := cli.Post(srv.buildURL(agentID, "checkin"), "application/json", bytes.NewBuffer([]byte("{}"))) //nolint:noctx // test case
```

Comment:
found a bug with the integration tests here. the connection is refused (error wasn't checked before), is that expected @marclop? or should we be logging the error via `t.Log()`?